jobs/redis-blacksmith-plans/templates/plans/standalone-6/manifest.yml

```diff
@@ -1,9 +1,25 @@
 ---
 meta:
   size: default
+  password: (( vault $CREDENTIALS "/redis/system:password" ))
+  environment: <%= p('environment') %>
+  bosh_env: <%= p('bosh_env') %>
   default:
     azs: [ z1 ]
-  password: (( vault $CREDENTIALS "/redis/system:password" ))
+
+variables:
+- name: redis_standalone_crt
+  type: certificate
+  options:
+    ca: (( concat "/" meta.bosh_env "-bosh/" meta.environment "-blacksmith/blacksmith_services_ca" ))
+    common_name: (( concat meta.environment "-redis-standalone.bosh" ))
+  consumes:
+    alternative_name: 
+      from: redis-san 
+      properties: { wildcard: true }
+
+features:
+  use_dns_addresses: true
```

Comment:
@itsouvalas same here.

Reply:
It depends on how the links are provided and consumed, how the director is configured and what redis is expecting. Generally, redis can handle both IP and URL based connections. As far as the certificate goes, it already uses `ip_addresses: false` which if available, it will return the corresponding (dns) address. You'd have to check the validity of the certificate created and proceed accordingly.